docs/libraries/http/reference/decorators.md

```diff
@@ -297,14 +297,14 @@ it will be used as a prefix to the route URI of the operation.
 
 #### Target
 
-`union Namespace | Interface | Operation`
+`Namespace | Interface | Operation`
 
 #### Parameters
 
-| Name    | Type                      | Description                                                                                                                                  |
-| ------- | ------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------- |
-| path    | `valueof scalar string`   | Relative route path. Cannot include query parameters.                                                                                        |
-| options | `model (anonymous model)` | Set of parameters used to configure the route. Supports `{shared: true}` which indicates that the route may be shared by several operations. |
+| Name    | Type             | Description                                                                                                                                  |
+| ------- | ---------------- | -------------------------------------------------------------------------------------------------------------------------------------------- |
+| path    | `valueof string` | Relative route path. Cannot include query parameters.                                                                                        |
+| options | ``               | Set of parameters used to configure the route. Supports `{shared: true}` which indicates that the route may be shared by several operations. |
```

Comment:
The empty string in place of anonymous model seems like it's a bit less descriptive than before

Reply:
yeah, it was because I made a few things use the same api to resolve type reference it it felt like `(anonymous model)` was bad in some other scenarios, I don't feel like its much better than an empty string, I think what we really need is the better anonymoius model renderer